packages/components/src/components/RadioCard/RadioCard.tsx

```diff
@@ -0,0 +1,55 @@
+import { ReactNode } from 'react';
+
+import styled, { css } from 'styled-components';
+
+import { palette, borders } from '@trezor/theme';
+
+import { Card } from '../Card/Card';
+import { Icon } from '../Icon/Icon';
+import { Box } from '../Box/Box';
+
+export type RadioCardProps = {
+    isActive: boolean;
+    children: ReactNode;
+    onClick: () => void;
+};
+
+const BorderActive = styled.div<{ isActive: boolean }>`
+    pointer-events: none;
+    position: absolute;
+    inset: 0;
+    ${({ isActive }) =>
+        isActive &&
+        css`
+            outline: ${borders.widths.large} solid ${({ theme }) => theme.borderSecondary};
+            outline-offset: -${borders.widths.large};
+        `}
+    border-radius: ${borders.radii.md};
+    z-index: 1;
+`;
+
+const IconWrapper = styled.div`
+    border-radius: ${borders.radii.full};
+    background-color: ${({ theme }) => theme.borderSecondary};
+    padding: 2px;
+`;
+
+export const RadioCard = ({ isActive, children }: RadioCardProps) => (
```

Comment:
nitpick: It would be nice to add also `frameProps`, but we can do it later on our own.